akka-docs/src/main/paradox/common/cluster.md

```diff
@@ -12,6 +12,12 @@ Akka Cluster provides a fault-tolerant decentralized peer-to-peer based cluster
 [membership](#membership) service with no single point of failure or single point of bottleneck.
 It does this using [gossip](#gossip) protocols and an automatic [failure detector](#failure-detector).
 
+Akka cluster allows for building distributed system, where a system spans multiple nodes. It is not intended as a 
+general communication between separate systems. See the discussion on 
```

Comment:
I'm fine with this terminology of "system", but I can imagine that some might misinterpret it.

ActorSystem vs system.

Not sure how to phrase it, perhaps "system (service)" ?

Reply:
There are no hard rules in the lagom docs but we try to use `application` as a sum of `services` perceived as one by the user. Then a `service` is a sum of `nodes` acting as one (aka, an akka cluster).

When using `system` I think we try to call it `Lagom system` and that'd be the equivalent to `application`. 

I like how you insist on stating the communication is only meant for intra-service. 👍